roles/openshift_cli/tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- openshift_facts:
+    role: common
+    local_facts:
+      cli_image: "{{ osm_image | default(None) }}"
```

Comment:
Is it possible to move this to `playbooks/common/openshift-cluster/initialize_facts.py`?  It seems to me that osm_image is coming from the inventory and we could just assign this to the rest of the facts we are setting in a single place.  

@michaelgugino, thoughts on this?